### ISSUE TITLE
Scenario citations- not all citations are being displayed on report

### ISSUE THREAD

**Issue by jknewson** [USGS-WiM/StreamStats#478]
[Low flow scenario](http://wim.usgs.gov/nssservices/regions/MA/scenarios?statisticgroups=lfs&unitsystems=2), has [2 citations](http://wim.usgs.gov/nssservices/citations?regressionregions=69,180), only first is being shown on report

![image](https://cloud.githubusercontent.com/assets/8713714/14918066/3566bf34-0ddf-11e6-8d77-a1c88ce25aee.png)

**Comment by marsmith** [comment]
according to the code, it should be looping over them (still researching):

```
                        <h4>{{StatisticsGroup.Name}} Citations</h4>
                        <div ng-repeat="citation in StatisticsGroup.Citations">
                            <a href="{{citation.CitationURL}}" target="_blank">{{citation.Author}}{{citation.Title}}</a>
                        </div>
```

**Comment by marsmith** [comment]
Yea, I only see one citation for screenshot above

**Comment by marsmith** [comment]
fixed in #516
